src/App/Datums.cpp

```diff
@@ -401,3 +387,10 @@ bool LocalCoordinateSystem::LCSExtension::extensionGetSubObject(DocumentObject*&
         return false;
     }
 }
+
+bool LocalCoordinateSystem::hasObject(const DocumentObject* obj, [[maybe_unused]] bool recursive) const
+{
+    auto* lcs = dynamic_cast<const LocalCoordinateSystem*>(getExtendedObject());
+    const auto& features = lcs->OriginFeatures.getValues();
```

Comment:
I'd prefer not using `auto` here. Idk if this is QList, std::vector, std::array... of course those experienced with the codebase knows, but `auto` should be used to improve readability, not the opposite6

Reply:
We should come up with good guidelines for it, as without it everyone will have different taste. Can we agree for now to leave it as is to not change back and forth and enforce it after we agree on the scope?